src/components/rockets/Rockets.js

```diff
@@ -0,0 +1,30 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import Rocket from './Rocket';
+import { showRockets } from '../../redux/rockets/action-makers';
+
+const RocketExihbit = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets);
+  useEffect(() => {
+    dispatch(showRockets());
+  }, [dispatch]);
```

Comment:
Hi @basem909 Would like to suggest adding ```store.dispatch(showRockets())```  in the configureStore.js;  as to make your code DRY and more optimized. Because the code below that already is using thunk to handle any side effects and catch errors upon your network request/APIs calls. 

 ```export const showRockets =() => async (dispatch) => {
    const rockets = await getApi();
     dispatch({ type: rocketShowed, payload: rockets });
  };
```